packages/protocol-http/src/Fields.ts

```diff
@@ -1,7 +1,7 @@
-import { Field, FieldOptions } from "./Field";
+import { Field } from "./Field";
```

Comment:
Since Field/Fields will be unused you can leave in the latest version if you want